contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,46 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* KEY COLUMN USAGE
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.key_column_usage
+AS SELECT CAST(sys.db_name() AS sys.nvarchar(128)) AS "CONSTRAINT_CATALOG",
+    CAST((select orig_name from sys.babelfish_namespace_ext where ss.nc_nspname = nspname) AS sys.nvarchar(128)) AS "CONSTRAINT_SCHEMA",
+    CAST(ss.conname AS sys.nvarchar(128)) AS "CONSTRAINT_NAME",
+    CAST(sys.db_name() AS sys.nvarchar(128)) AS "TABLE_CATALOG",
+    CAST((select orig_name from sys.babelfish_namespace_ext where ss.nr_nspname = nspname)  AS sys.nvarchar(128)) AS "TABLE_SCHEMA",
+    CAST(ss.relname AS sys.nvarchar(128)) AS "TABLE_NAME",
+    CAST(a.attname AS sys.nvarchar(128)) AS "COLUMN_NAME",
+    CAST((ss.x).n AS sys."int") AS "ORDINAL_POSITION",
+        CAST(CASE
+            WHEN ss.contype = CAST('f' AS sys.bpchar) THEN information_schema._pg_index_position(ss.conindid, ss.confkey[(ss.x).n])
+            ELSE CAST(NULL AS integer)
+        END AS information_schema.cardinal_number) AS "POSITION_IN_UNIQUE_CONSTRAINT"
+   FROM pg_attribute a,
+    ( SELECT r.oid AS roid,
+            r.relname,
+            r.relowner,
+            nc.nspname AS nc_nspname,
+            nr.nspname AS nr_nspname,
+            c.oid AS coid,
+            c.conname,
+            c.contype,
+            c.conindid,
+            c.confkey,
+            c.confrelid,
+            information_schema._pg_expandarray(c.conkey) AS x
+           FROM pg_namespace nr,
```

Comment:
We should use pg_namespace_ext(joined by babelfish_namespace_ext) instead.

Reply:
Resolved in latest commit.